docs/learn-identity.md

```diff
@@ -22,11 +22,11 @@ information, so the data must be less than that. When inputting the data manuall
 The easiest way to add the built-in fields is to click the gear icon next to one's account and
 select "Set on-chain identity".
 
-![Gear icon provides the option to set identity](/img/identity/01.jpg)
+![Gear icon provides the option to set identity](assets/identity/01.jpg)
```

Comment:
Could you update these images? they are still the old ones from the wiki before the UI update